snippets/properties.json

```diff
@@ -0,0 +1,23 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1|jdbc:mariadb://localhost:3306/mydb,jdbc:mysql://localhost:3306/test,jdbc:h2:mem:mydb,jdbc:postgresql://localhost/mydb,jdbc:sqlserver://localhost:1433;databaseName=mydb|}",
+			"quarkus.datasource.driver=${2|org.mariadb.jdbc.Driver,org.mysql.jdbc.Driver,org.h2.Driver,org.postgresql.Driver,com.microsoft.sqlserver.jdbc.SQLServerDriver|}",
```

Comment:
that's great it works! What about other JDBC driver like derby? Perhaps we will add another JDB driver according user feedback?

Reply:
I think Derby is not supported yet in Quarkus.

Reply:
Thanks @lordofthejars for your feedback! We will add another database according user feedback.

Reply:
I listed the drivers according to the quarkus extensions found in https://quarkus.io/guides/datasource-guide#jdbc-url-configurations